samples/MASMessagingSample/build.gradle

```diff
@@ -33,10 +33,13 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    testCompile 'junit:junit:4.12'
     compile "com.android.support:recyclerview-v7:$rootProject.ext.recyclerViewVersion"
     compile "com.android.support:design:$rootProject.ext.designVersion"
+    compile 'pl.droidsonroids.gif:android-gif-drawable:1.2.7'
 
     compile project(':masui')
     compile project(':mas-connecta')
+    compile project(':mas-identity-management')
+    compile 'com.android.support.constraint:constraint-layout:1.0.2'
```

Comment:
In the Android-MAS-SDK root project's build.gradle, there's a section for library versions so we can keep them consistent across sample apps. Could you update the `ext{}` section in the file to include the constraint layout library?